dist/profile/manifests/confluence.pp

```diff
@@ -22,7 +22,9 @@
   }
   file { '/srv/wiki/home':
     ensure  => directory,
-    recurse => true,
+    # confluence container is baked with UID=1000 & GID=1001
+    owner   => 1000,
```

Comment:
How will you be sure that these aren't used already on any of the sytems that this Puppet might run on? 1000 is a pretty low and most likely is already used.

On my personal Linux machine, 1000 is the UID of the first user